src/layouts/default.vue

```diff
@@ -42,12 +42,21 @@ export default {
 
   computed: {
     routes() {
-      return [
-        '/General',
-        '/PortForwarding',
-        '/Images',
-        '/Troubleshooting',
+      const routes = [
```

Comment:
I would feel better if this object didn't have the same name as the enclosing method (I know it's allowed, but I think we can be clearer). Maybe `routeTable` instead of `routes`?

Reply:
I think that's a fair assessment, fixed with your suggestion 🙂